### ISSUE TITLE
Attach to lib.rs does not notice attributes in lib.rs.

### ISSUE THREAD

**Issue by smj389** [intellij-rust/intellij-rust#5934]
## Environment

* **IntelliJ Rust plugin version:** 0.3.128.3278-202
* **Rust toolchain version:** 1.47.0-nightly (81dc88f88 2020-08-13) x86_64-pc-windows-msvc
* **IDE name and version:** CLion 2020.2 (CL-202.6397.106)
* **Operating system:** Windows 10 10.0

## Steps to reproduce

1. Imagine a lib.rs file like this:

```rust
#![allow(dead_code)]
#![feature(async_closure)]
mod one;
mod two;

```
2. Create a new file in the crate, e.g. three.rs
3. Plugin suggests "attach to lib.rs"
4. Get following result:
```rust
mod three;

#![allow(dead_code)]
#![feature(async_closure)]
mod one;
mod two;

```

5. Problem is, of course, that the attributes need to be at the top of the file.
6. 😢

**Comment by Undin** [comment]
@Kobzol I think it can be fixed together with #5903

Note, we already have similar logic in auto-import quick fix. Probably, we can reuse it

**Comment by Undin** [comment]
@smj389 Thanks for the report!
